runtime-modules/project-token/src/lib.rs

```diff
@@ -214,6 +215,66 @@ decl_module! {
         /// Predefined errors.
         type Error = Error<T>;
 
+        /// Allow Governance to Set constraints
+        /// Preconditions:
+        /// - origin is signed by `root`
+        /// PostConditions:
+        /// - governance parameters storage value set to the provided values
+        /// <weight>
+        ///
+        /// ## Weight
+        /// `O (1)`
+        /// # </weight>
+        #[weight = WeightInfoToken::<T>::update_governance_parameters()]
```

Comment:
the name change is not reflected in the function.
```suggestion
        #[weight = WeightInfoToken::<T>::update_token_constraints()]
```